### PR TITLE
Fix issue #3133

### LINKED ISSUE #3133: Fix typos in the Developer Guidelines
### Please confirm the following

- [X] I checked [the current issues](https://github.com/Ivy-Apps/ivy-wallet/issues) for duplicate issues.

### What would you like to improve?

Read our Dev guidelines and its pages. Fix any typos and mistakes. 
https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md

### Because...

- Better dev docs

### Description

_No response_

### Success Criteria

_No response_

### PR DESCRIPTION
**Pull request by biplab1**
## Pull Request (PR) Checklist
Please check if your pull request fulfills the following requirements:
- [x] The PR is submitted to the `main` branch.
- [x] I've read the [Contribution Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/CONTRIBUTING.md) and my PR doesn't break the rules.
- [x] I've read and understand the [Developer Guidelines](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/Guidelines.md).
- [ ] I confirm that I've run the code locally and everything works as expected.
- [ ] 🎬 I've attached a **screen recoding** of the changes. 

> Tip: drag & drop the video to the PR description.

## What's changed?

Describe with a few bullets **what's new:**
- Fixed typos and mistakes in Dev guidelines and its pages, excluding Yoda-Style-Architecture.md

> 💡 Tip: Please, attach screenshots and screen recordings. It helps a lot!

## Risk Factors

**What may go wrong if we merge your PR?**

- a
- b

**In what cases your code won't work?**

- a
- b

## Does this PR closes any GitHub Issues?

Check **[Ivy Wallet Issues](https://github.com/Ivy-Apps/ivy-wallet/issues)**.

- Closes #3133

> Replace `ISSUE_NUMBER` with the number of the GitHub issue that this PR is fixing. If you've done that correctly, you'll see the issue title linked when previewing your PR description.

> **!Note:** Do **not** link the PR number. Link the number/id of the GitHub Issue from [issues](https://github.com/Ivy-Apps/ivy-wallet/issues).

## Troubleshooting CI failures ❌

GitHub Actions failing? Read our [CI Troubleshooting guide](https://github.com/Ivy-Apps/ivy-wallet/blob/main/docs/CI-Troubleshooting.md).

### CODE REVIEW

File: .idea/.name
Comment (ILIYANGERMANOV) [inline .idea/.name:1]: Revert that change

File: .idea/checkstyle-idea.xml
Comment (ILIYANGERMANOV) [inline .idea/checkstyle-idea.xml:1]: Revert that change. Considering adding it to `.gitignore`

**Review by ILIYANGERMANOV — CHANGES_REQUESTED** [review]
Amazing work!! The docs looks much more readable and nicer after your contribution 👏 
Please revert the `.idea` file changes and we can merge

**Comment by biplab1** [comment]
Thank you for the kind words! I'm glad to hear that the documentation improvements are appreciated.

I've reverted the .idea file changes as requested. The pull request should now be ready for merging.

Please let me know if there's anything else needed.

**Review by ILIYANGERMANOV — APPROVED** [review]
Excellent work 👏

File: .idea/.name
Comment (biplab1) [reply .idea/.name:1]: Reverted the .idea/.name change.

File: .idea/checkstyle-idea.xml
Comment (biplab1) [reply .idea/checkstyle-idea.xml:1]: Reverted the change. Added it to `.gitignore`.

**Comment by biplab1** [comment]
Thank you! I'm glad you liked it.

**Pull request merged**